pandas/core/generic.py

```diff
@@ -10690,9 +10690,13 @@ def _add_series_or_dataframe_operations(cls):
         the doc strings again.
         """
 
-        from pandas.core import window as rwindow
+        from pandas.core.window import (
+            ewm as lib_ewm,
```

Comment:
we usually call the _libs imports like libwindow (etc), so is there a reason you can't just
```from pandas.core.window import ewm, expanding, rolling```? (or else I guess would be ok to call libewm, etc)

Reply:
or just directly import what you need from these modules?

Reply:
This caused a `RecursionError` since the defined functions here are:

```
def rolling(...):
    return rolling(...)

cls.rolling = rolling
```

Reply:
Okay I was able to use more direct imports, and all green.